### PR TITLE
Add new translation strings + Update PT-BR language pack

### LINKED ISSUE #705: translation for custom bandwith monitor in zh_cn
some translate fix fo zh_cn.
(I can't make a pull request with a single commit,so only as  pathc here)

[fix_translate_in_zh-cn.txt](https://github.com/ericpaulbishop/gargoyle/files/1584549/fix_translate_in_zh-cn.txt)

### PR DESCRIPTION
**Pull request by d3fz**
- Add new PT-BR/ZH-CN translation strings to the newly added "Custom Bandwidth Monitor" feature. (Closes #705 )
- Minor updates and fixes to the Portuguese language pack.

### CODE REVIEW

File: package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/update.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ upS.SelF="Selecione o firmware";
 upS.Prsv="Manter Configurações (<b>não recomendado</b>)";
 
 //javascript
-upS.SelErr="ERRO: Você deve selecionar um arquivo de firmware.";
-upS.Warn2="A atualização de firmware apagará completamente suas configurações, e pode, eventualmente, inutilizar (&quot;brickar&quot;) seu roteador se o arquivo de firmware selecionado for inválido. Deseja continuar ?";
+// upS.SelErr="ERRO: Você deve selecionar um arquivo de firmware.";
```
Comment (lantis1008) [inline package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/update.js]: Are you sure you want to comment this one out? It is used in the update page

Comment (d3fz) [reply package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/update.js]: Not at all. I was testing something real quick and apparently forgot to uncomment afterward. 

Fixed, thanks for pointing it out.

**Comment by d3fz** [comment]
Hi @lantis1008 ,

I currently have another change ready that I'd like to PR, but is not related to this one. It's also on another branch. Should I open a new PR or just merge with this one and update the title? I wouldn't like to get this PR fairly larger with unrelated things.

Thanks in advance.

**Comment by lantis1008** [comment]
If it’s already on a separate branch, and it’s unrelated to this one, I’d go a separate PR. 

I tend to bring them all into one PR because I work on a single branch for everything (out of laziness).

**Comment by d3fz** [comment]
Understood. Speaking of related things, I could probably add the recent Chinese (ZH-CN) translation request to this one, so it goes altogether. What do you say?

I'll open a new PR for the other change then.

Merry Christmas and Happy Holidays!

**Comment by lantis1008** [comment]
That would be great. Saves me doing it. I’m with family until tomorrow. 

Merry Christmas to you too. Appreciate your help throughout the year.

**Pull request merged**